trulens_eval/trulens_eval/feedback/provider/hugs.py

```diff
@@ -187,6 +188,48 @@ def get_scores(text):
 
         return l1, dict(text1_scores=scores1, text2_scores=scores2)
 
+    @_tci
+    def context_relevance(self, prompt: str, context: str) -> float:
```

Comment:
Should include some message on how to get access to truera/context_relevance model

Reply:
done